### PR TITLE
Original Gem boilerplate.

### PR DESCRIPTION
**Pull request by akilmadan**

### CODE REVIEW

**Comment by msmirnoff** [comment]
@vgmaster21 

I don't think we need to PR the initial commit :smile:

**Comment by akilmadan** [comment]
@msmirnoff This is more of a 'second commit' with all the initial Gem files setup (including Readme). Would be worth a quick scan and review.

File: README.md [hunk 1]
```diff
@@ -1 +1,61 @@
...
+
+## Usage
+
+Create a list of Arbritrary objects
+'''ruby
+elements = [Employee.new(name: 'Mr Smith', age: 30), Employee.new(name: 'Mrs Wilson ', age: 45)]
+'''
+
+Now define the attributes of that object which are queryable
+Note that the object mustb e able to respond when called with one of these attributes
```
Comment (msmirnoff) [inline README.md]: Typo?

File: queryable_collection.gemspec [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'queryable_collection/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'queryable_collection'
+  spec.version       = QueryableCollection::VERSION
+  spec.authors       = ['Akil Madan']
+  spec.email         = ['akil.madan@outlook.com']
```
Comment (msmirnoff) [inline queryable_collection.gemspec]: I assume you are happy to make this public?

**Comment by msmirnoff** [comment]
@vgmaster21 
It's all good, a couple of minor comments.

**Comment by mnoack** [comment]
:+1: I've addressed @msmirnoff's comments which I agreed with them, I've also added some travis tweaks I use as well.

**Comment by mnoack** [comment]
Merge if you're happy with my changes, and rebase the other one if you need to

**Pull request merged**